buildout_1.rst

```diff
@@ -16,9 +16,9 @@ Topics covered:
 
 .. only:: not presentation
 
-    `Buildout <https://pypi.python.org/pypi/zc.buildout>`_ composes your application for you, according to your rules.
+    `Buildout <https://pypi.python.org/pypi/zc.buildout>`_ configures your application for you, according to your rules.
 
-    To compose your application you must define the eggs you need, which version, what configuration files Buildout has to generate for you, what to download and compile, and so on.
+    To configure your application you must define the eggs you need, which version, what configuration files Buildout has to generate for you, what to download and compile, and so on.
     Buildout downloads the eggs you requested and resolves all dependencies. You might need five different eggs, but in the end, Buildout has to install 300 eggs, all with the correct version in order to resolve all the dependencies.
```

Comment:
IIRC I wrote this section with the wording `composes` and it catches the actions performed by buildout better than `configures`.
Quite often, buildout does not only generate configuration files but compiles nginx and so on. 
Even if it would only provide configuration, the fact that it tells plone, where to find the zeo server, where the fronted web servers find the backend and so on is more an action of composing the system than just a simple configuring an application.